src/som/VM.java

```diff
@@ -46,8 +46,9 @@
 import som.vmobjects.SClass;
 import som.vmobjects.SObjectWithClass.SObjectWithoutFields;
 import som.vmobjects.SSymbol;
-import tools.concurrency.ActorExecutionTrace;
+import tools.concurrency.MedeorTrace;
```

Comment:
MedeoTrace?

This seems to be the wrong name.
My understanding is that the "Debugger" elements in the interpreter are called Medeor, so, that should perhaps be a tools.medeor package.
However, the trace here, refers probably to the kompos protocol, and as such, should probably name after the protocol and not the subsystem implementing it.